src/worldofzuul/Game.java

```diff
@@ -4,15 +4,37 @@
  * @author  Michael Kolling and David J. Barnes
  * @version 2006.03.30
  */
+
 public class Game {
+    /** 
+     * There is two attributes, parser and currentRoom.
+     * 
+     * The two attributes parser and currentRoom are instances from the classes
+     * Parser and Room. The currentRoom variable is what room the player is 
+     * in at the current moment. 
+     * 
+     */
     private Parser parser;
     private Room currentRoom;
```

Comment:
Skal have sin egen docblock

Reply:
fixed in: 6300f68b92537827114d8b1698fabe9b6ba99f28